internal/common/common.go

```diff
@@ -45,12 +45,10 @@ ipv6.dhcp-duid=ll
 path=/etc/NetworkManager/system-connections-merged
 `
 
-// Configuration of the NM hostname mode in case of static network configuration
-// needed in order to prevent hostname update in bootstrap node, which may cause dns resolution
-// failure due to OCPBUGSM-26430
-const StaticNetworkHostnameConf = `
+// configuration of NM to disable handling of /etc/resolve.conf
```

Comment:
resolv.conf

Reply:
maybe enlarge the comment? this is relevant only to bootstrap before reboot right?

Reply:
fixed